### PR TITLE
Update KaimingUniform & KaimingNormal initializer docs

### PR DESCRIPTION
**Pull request by Ray2020BD**
Update KaimingUniform & KaimingNormal initializer docs
image
image
image
image

### CODE REVIEW

File: doc/paddle/api/paddle/nn/initializer/kaiming/KaimingUniform_cn.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    在大多数情况下推荐设置fan_in为None
+
+**代码示例**：
+
+.. code-block:: python
+
+    import paddle
+    import paddle.nn as nn
+    paddle.disable_static()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/initializer/kaiming/KaimingUniform_cn.rst]: 不需要这一句了

File: doc/paddle/api/paddle/nn/initializer/kaiming/KaimingNormal_cn.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    在大多数情况下推荐设置fan_in为None
+
+**代码示例**：
+
+.. code-block:: python
+
+    import paddle
+    import paddle.nn as nn
+    paddle.disable_static()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/initializer/kaiming/KaimingNormal_cn.rst]: 不需要这一句了

File: doc/paddle/api/paddle/nn/initializer/kaiming/KaimingUniform_cn.rst (hunk 1 above)
Comment (Ray2020BD) [reply doc/paddle/api/paddle/nn/initializer/kaiming/KaimingUniform_cn.rst]: deleted

File: doc/paddle/api/paddle/nn/initializer/kaiming/KaimingNormal_cn.rst (hunk 2 above)
Comment (Ray2020BD) [reply doc/paddle/api/paddle/nn/initializer/kaiming/KaimingNormal_cn.rst]: deleted

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**